airbyte-integrations/bases/base-normalization/integration_tests/dbt_integration_test.py

```diff
@@ -59,6 +59,8 @@ def setup_db(self, destinations_to_test: List[str]):
             self.setup_clickhouse_db()
         if DestinationType.TIDB.value in destinations_to_test:
             self.setup_tidb_db()
+        if DestinationType.DUCKDB.value in destinations_to_test:
```

Comment:
I'd be curious if you just don't have these two lines (+ delete the setup_duckdb_method) if tests still work. I think that would reduce test runtime, especially for local developers

Reply:
I initially tried to remove, but I did it wrong, which resulted in this [error](https://github.com/airbytehq/airbyte/pull/22528#issue-1575062145). But I [removed](https://github.com/airbytehq/airbyte/pull/22566/commits/37c4e2cd6df5f3b1f0557809e9e4096ddf046023) it now as suggested, and the tests are all green still. Will resolve this convo.